addons/website_product_comparison/models/product_comparison.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from collections import OrderedDict
+
+from openerp import api, fields, models
+
+
+class ProductAttributeCategory(models.Model):
+    _name = "product.attribute.category"
+    _description = "Product Attribute Category"
+    _order = 'sequence'
+
+    name = fields.Char("Category Name", required=True)
+    sequence = fields.Integer("Sequence")
+
+
+class ProductAttribute(models.Model):
+    _inherit = 'product.attribute'
+
+    category_id = fields.Many2one('product.attribute.category', string="Category")
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    def get_variant_groups(self):
+        res = OrderedDict()
+        for var in self.attribute_line_ids:
+            res.setdefault(var.attribute_id.category_id.name or 'Uncategorized', []).append(var)
+        return res
+
+
+class ProductWishlist(models.Model):
+    _name = 'product.wishlist'
+
+    partner_id = fields.Many2one('res.partner', string='Customer', required=True)
+    pricelist_id = fields.Many2one('product.pricelist', string='Pricelist when added')
+    currency_id = fields.Many2one('res.currency', related='pricelist_id.currency_id', readonly=True)
+    website_id = fields.Many2one('website', required=True)
+    price = fields.Monetary(digits=0, currency_field='currency_id')
+    price_new = fields.Float('Price', compute='compute_new_price')
```

Comment:
These two are to differentiate the price when added to the wishlist and the current price but not something customisable? A tooltip/better string would be nice.
Is it possible to use a Monetary field too to be consistent?